cas/cas.h

```diff
@@ -400,6 +400,16 @@ struct m0_cas_op {
 	struct m0_dtm0_tx_desc cg_txd;
 } M0_XCA_RECORD M0_XCA_DOMAIN(rpc);
 
+/**
+ * The structure to be passed in DTM0 log as a payload
+ */
+struct m0_cas_dtm0_log_payload  {
+	/** CAS op */
+	struct m0_cas_op cdg_cas_op;
+	/** CAS operation code */
```

Comment:
Please add comments here to explain:
This is the rpc fop/item opcode:
M0_CAS_DEL_FOP_OPCODE or M0_CAS_PUT_FOP_OPCODE

because there is another enum called:
```
347 enum m0_cas_opcode {
348         CO_GET,
349         CO_PUT,
350         CO_DEL,
351         CO_CUR,
```